corehq/apps/consumption/shortcuts.py

```diff
@@ -17,39 +15,32 @@ def get_default_monthly_consumption(domain, product_id, location_type, case_id):
     parameters.
     """
 
-    keys = [
```

Comment:
@dannyroberts This function is the couch question I had for you. My understanding of it is that using an array of arrays of keys returns all documents that match the first key, then all that match the second, and so on.